_posts/2019-12-18-sentry-the-guardian-of-your-quarkus-app.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: 'Sentry Error Tracker - The new guardian of your Quarkus application'
```

Comment:
minor nit but due to sentry's recent license changes it rings wrong to me that we call this "*the* new guardian of your quarkus application" ....could we tone it down to for example "A guardian of your Quarkus application" . ?